config.lua

```diff
@@ -0,0 +1,5 @@
+Config = Config or {}
+
+Config.UseNoName = true -- Dont use any name when someone wins
+Config.UseCharName = false -- Use the character name when announcement is made on win
```

Comment:
```suggestion
Config.UseCharName = false -- Uses the character name if true, if false it uses the steam / default name.
```
The explenation for this line is quite incorrect, this explenation would fit it alot better.